colibris/views/api.py

```diff
@@ -44,22 +56,22 @@ def get_query_schema(self, *args, **kwargs):
 
         return data
 
-    async def get_request_body(self):
+    def get_request_body(self):
         if not self.request.body_exists:
             return {}
 
         try:
-            json_payload = await self.request.json()
+            json_payload = json.loads(self.request[BODY_KEY])
```

Comment:
One last small remark: in `authentication`, the account request key is called `_REQUEST_ACCOUNT_ITEM_NAME`. I'm not insisting on using any of these two patterns (`KEY` vs `ITEM_NAME`) but I want consistency. What do you prefer?